src/nvim/ascii.h

```diff
@@ -87,4 +90,45 @@
 # define PATHSEPSTR     "/"
 #endif
 
+static inline bool ascii_iswhite(int c) FUNC_ATTR_ALWAYS_INLINE FUNC_ATTR_CONST;
+static inline bool ascii_isdigit(int c) FUNC_ATTR_ALWAYS_INLINE FUNC_ATTR_CONST;
+static inline bool ascii_isxdigit(int c) FUNC_ATTR_ALWAYS_INLINE FUNC_ATTR_CONST;
+static inline bool ascii_isspace(int x) FUNC_ATTR_ALWAYS_INLINE FUNC_ATTR_CONST;
+
+/// ascii_iswhite() is used for "^" and the like. It differs from isspace()
+/// because it doesn't include <CR> and <LF> and the like.
+static inline bool ascii_iswhite(int c)
+{
+  return c == ' ' || c == '\t';
+}
+
+/// Use our own isdigit() replacement, because on MS-Windows isdigit() returns
+/// non-zero for superscript 1.  Also avoids that isdigit() crashes for numbers
+/// below 0 and above 255.
+static inline bool ascii_isdigit(int c)
+{
+  return c >= '0' && c <= '9';
+}
+
+/// Variant of isxdigit() that can handle characters > 0x100.
+/// We don't use isxdigit() here, because on some systems it also considers
+/// superscript 1 to be a digit.
+///
+/// @param c
+/// @return TRUE if the character is a hexadecimal digit.
```

Comment:
`TRUE` -> `true`, and perhaps c doesn't need to be mentioned if there's no description (as is done for the other functions).